consoleApp/src/main/java/by/issoft/consoleApp/handlers/MyHandler.java

```diff
@@ -0,0 +1,11 @@
+package by.issoft.consoleApp.handlers;
+
+import by.issoft.store.Store;
+
+public interface MyHandler {
+    /**
+     * Logic for command execution
+     * @param store info for command execution
+     */
+    void execute(Store store);
```

Comment:
From my point of view no need to do this because _top_, _sort_ and _quit_ commands may return list of products but _print_ can't return list of products while print store. 
To implement such thing may produce lots of code, building difficult hierarchies.

Reply:
i was talking about returning string. but, dont worry, it is a minor improvement.